### PR TITLE
Avoid opening a new tab when a link redirects to an internal organization page

### PR DESCRIPTION
**Pull request by jparelladab**
I've added some jquery in application.js to look for each anchor tag, and remove target=_blank IF it's an internal hiperlink (it points to https://participa.challenge.cat/...)

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -13,3 +13,13 @@
 //= require rails-ujs
 //= require_tree .
 //= require decidim
+
+//remove target=_blank from each anchor tag with internal href
+$(document).ready(function() {
+	$('a').each(function(){
+	  const link = this;
+	  if (link.href.includes("https://participa.challenge.cat")) {
```
Comment (victorol1) [inline app/assets/javascripts/application.js]: You should remove `https:// `from the string since it's not needed for the URL to be valid.

**Comment by victorol1** [comment]
@jparelladab try to put in the description any related links that explain the issue, if there's any.

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (jparelladab) [reply app/assets/javascripts/application.js]: Thank you @victorol1 . Done.

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -13,3 +13,13 @@
...
 //= require decidim
+
+//remove target=_blank from each anchor tag with internal href
+$(document).ready(function() {
+	$('a').each(function(){
+	  const link = this;
+	  if (link.href.includes("participa.challenge.cat")) {
+	    $(this).removeAttr("target");
+	  }
+	});
```
Comment (jesusdb) [inline app/assets/javascripts/application.js:24]:
```suggestion
  $('a').each(function(){
    const link = this;
    if (link.href.includes('participa.challenge.cat')) {
      $(this).removeAttr('target');
    }
  });
```

File: app/assets/javascripts/application.js [hunk 3]
```diff
@@ -13,3 +13,13 @@
 //= require rails-ujs
 //= require_tree .
 //= require decidim
+
+//remove target=_blank from each anchor tag with internal href
```
Comment (jesusdb) [inline app/assets/javascripts/application.js:17]:
```suggestion
/**
 * This removes the attribute `target` from each anchor tag
 * when the URL redirects to an external page.
 */
```

**Pull request merged**